lib/opensusebasetest.pm

```diff
@@ -914,17 +914,6 @@ sub handle_broken_autologin_boo1102563 {
     wait_screen_change { send_key 'alt-f4' };
 }
 
-sub handle_additional_polkit_windows_bsc1157928 {
```

Comment:
We also should remove place where we call this method. AFAIR that method is called not on shutdown, but on first boot. So we might want to just update bugreference or use this workaround in both places as was proposed as an idea in the ticket.

Reply:
Of course, how could I forget that... I removed it, it shouldn't be called anywhere now :
```
jonathan@/var/lib/openqa/share/tests/sle/tests  (fix_shutdwn)$ grep -r handle_additional_polkit_windows_bsc1157928 ..
jonathan@/var/lib/openqa/share/tests/sle/tests  (fix_shutdwn)$ 
```